src/components/SideBar/ItemDrawer/ItemDrawer.css

```diff
@@ -47,6 +47,6 @@
 }
 
 .ItemDrawer .overflow-container {
-  height: 100%;
+  height: calc(100% - 54px);
```

Comment:
What is this `54` accounting for? should we have a CSS variable in case it changes?

Reply:
It's the "Item drawer" header + list/grid switcher. I could store that height in a variable but I don't think we'd use it anywhere else than on this line.